### ISSUE TITLE
Layerfs problem

### ISSUE THREAD

**Issue by M-1-RLG** [LumaTeam/Luma3DS#432]
I know I had a issue like this before, About acnl:wa update not patching but turns out it's just the game I downloaded the actual game and it didn't patch the update stuff

**Comment by AuroraWright** [comment]
I'm not sure what you mean by that, but I checked the update and it mounts as patch:/, which Luma supports since 7.0.2.

**Comment by M-1-RLG** [comment]
Let me try again

**Comment by M-1-RLG** [comment]
Okay thanks it works!
